From 42baf7f9c25bdb81b3d83c9b184bbb082be4874b Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Sat, 7 Jul 2012 02:05:30 +0800
Subject: [PATCH 021/281] ENGR00215971 add property to config 2D dither
 feature

For the lower color precision display devices, need enable dither
to reduce obvious color grading effect, this feature is disabled default,
change init.freescale.rc as 'setprop hwc.enable_dither 1' to enable it

Signed-off-by: Xianzhong <b07117@freescale.com>
---
 imx6/sabresd/init.rc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/imx6/sabresd/init.rc b/imx6/sabresd/init.rc
index 3a107c7..3bd0a3b 100755
--- a/imx6/sabresd/init.rc
+++ b/imx6/sabresd/init.rc
@@ -18,8 +18,10 @@ on boot
     setprop debug.sf.showfps    0
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
+
     setprop hwc.filter_stretch  1
     setprop persist.sys.ui.hw   1
+    setporp hwc.enable_dither   0
 
     # deine board type
     setprop sys.devicy.type  "tablet"
-- 
1.8.0

